src/sentry/models/release.py

```diff
@@ -96,6 +96,9 @@ class ReleaseModelManager(BaseManager["Release"]):
     def get_queryset(self) -> ReleaseQuerySet:
         return ReleaseQuerySet(self.model, using=self._db)
 
+    def filter(self, *args, **kwargs) -> ReleaseQuerySet:
+        return self.get_queryset().filter(*args, **kwargs)
```

Comment:
ehhhh in theory we should have the django plugin do this -- there's just a microbug there

though I'd actually rather we factor out this custom queryset class.  it causes weird runtime differences too that are even harder to represent in the type system